src/fiatconnect/slice.ts

```diff
@@ -22,6 +28,8 @@ export interface State {
   quotesError: string | null
   transfer: FiatConnectTransfer | null
   providers: FiatConnectProviderInfo[] | null
+  recentlyUsedFiatAccounts: CachedFiatAccountUse[]
```

Comment:
nit: Seems like we use 3 different names for this `cachedFiatAccount`, `cachedFiatAccountUse`, `recentlyUsedFiatAccount`. Can we just use once for state item, selector and state item type